cmd/kyma/provision/aks/opts.go

```diff
@@ -1,4 +1,4 @@
-package gcp
```

Comment:
This is really weird... The GCP opts moved to AKS and the Azure options moved to GKE.
Are we sure they are identical?

Reply:
Yes, they are actually identical: https://github.com/kyma-project/cli/blob/master/cmd/kyma/provision/gcp/opts.go and https://github.com/kyma-project/cli/blob/master/cmd/kyma/provision/azure/opts.go